src/clients/asset-client.js

```diff
@@ -133,19 +145,22 @@ class AssetClient {
 
   __createOrUpdate (options = {}) {
     const fileName = options.file.name
-    const variations = Object.keys(options.variations).map(key => {
-      return {
-        [key]: JSON.stringify(options.variations[key])
-      }
-    })
+    const formats = Object.keys(options.formats).reduce((acc, key) => {
+      const format = options.formats[key]
+      format.in = JSON.stringify(format.in)
```

Comment:
`JSON.stringify` should not be necessary?

What about `format.ext`?

Reply:
right I forgot it's stringified on `asset-service` side

Reply:
>  What about `format.ext`?

You mean if it's not present?

Reply:
No, sorry my fault for misunderstanding the code.